arch/xtensa/src/esp32/esp32_irq.c

```diff
@@ -728,12 +738,13 @@ int esp32_cpuint_initialize(void)
    *   ESP32_CPUINT_PROFILING  11  Not yet defined
    *   ESP32_CPUINT_TIMER1     15  XTENSA_IRQ_TIMER1  1
    *   ESP32_CPUINT_TIMER2     16  XTENSA_IRQ_TIMER2  2
-   *   ESP32_CPUINT_SOFTWARE1  29  Not yet defined
+   *   ESP32_CPUINT_SOFTWARE1  29  XTENSA_IRQ_SWINT
```

Comment:
```suggestion
   *   ESP32_CPUINT_SOFTWARE1  29  XTENSA_IRQ_SWINT  4
```
Also, as `ESP32_IRQ_MAC` is the first peripheral interrupt, it will get shifted to the index 5, as defined here:
https://github.com/apache/incubator-nuttx/blob/master/arch/xtensa/include/esp32/irq.h#L195